apps/web/pages/auth/login.tsx

```diff
@@ -57,7 +57,10 @@ export default function Login({
         .string()
         .min(1, `${t("error_required_field")}`)
         .email(`${t("enter_valid_email")}`),
-      password: z.string().min(1, `${t("error_required_field")}`),
+      password: z
+        .string()
+        .min(1, `${t("error_required_field")}`)
+        .or(z.literal("")),
```

Comment:
Password can be empty when entering a TOTP code in the UI, which has the user/password form behind it getting the email prefilled from JWT token

Reply:
This would allow receiving password: ""? Is there a way to use if when case here?
// Pseudo
If (prop.disabledPassword or 2fa flag) {
password = z.string().min(1);
} else {
password = z.literal("");
}